app/src/main/java/com/bytemedrive/signup/EventCustomerSignedUp.kt

```diff
@@ -13,6 +14,10 @@ data class EventCustomerSignedUp(
 ) : Convertable {
 
     override suspend fun convert(database: ByteMeDatabase) {
-        database.customerDao().add(CustomerEntity(username, wallet, signedUpAt, null))
+        database.customerDao().add(
+            CustomerEntity(
+                username, wallet, signedUpAt, Customer.DEFAULT.balanceGbm, Customer.DEFAULT.dataUsageOverWiFi, thumbnailCreation = Customer.DEFAULT.thumbnailCreation
```

Comment:
use direct values instead this constant `DEFAULT`. Event should set exact values specified only inside event. Not somewhere outside.
```
    override suspend fun convert(database: ByteMeDatabase) {
        database.customerDao().add(CustomerEntity(username, wallet, signedUpAt, null, false, null))
    }
```